src/pool.rs

```diff
@@ -326,6 +326,21 @@ impl ConnectionPool {
 
             self.stats.server_tested(server.process_id(), address.id);
 
+            // Will return error if timestamp is greater than current system time, which it should never be set to
+            let require_health_check = server
+                .latest_successful_server_interaction_timestamp()
+                .elapsed()
+                .unwrap()
+                .as_millis()
+                > 10000; // TODO: Make this configurable
```

Comment:
need to make this a config

Reply:
You want to replace `latest_successful_server_interaction_timestamp` with `last_healthcheck_timestamp`.